recws.go

```diff
@@ -36,6 +37,7 @@ type RecConn struct {
 	// Proxy specifies the proxy function for the dialer
 	// defaults to ProxyFromEnvironment
 	Proxy func(*http.Request) (*url.URL, error)
+	TLSClientConfig  *tls.Config
```

Comment:
Please just add a comment to this property as all other properties

Reply:
done, please check